tests/test_custom_checkers/test_infinite_loop_checker.py

```diff
@@ -0,0 +1,177 @@
+import astroid
+import pylint.testutils
+
+from python_ta.checkers.infinite_loop_checker import InfiniteLoopChecker
+
+
+class TestInfiniteLoopChecker(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = InfiniteLoopChecker
+
+    def test_var_not_updated(self) -> None:
+        """Test that the checker correctly flags a while loop when no condition variables are used in the loop body."""
+        src = """
+            i, k, l = 0, 20, 40
+            j = 10
+            while i < 100 and k < 21 or l < 40:
```

Comment:
A technique that some tests use (but not all) is to add the comment `#@` to the end of the source code line where the node you want to extract begins. That would be the line with the `while` keyword in this case.

By doing so, you can then use `astroid.extract_node` to obtain exactly the node you want, without needing to call `nodes_of_class`.

Please update the tests here to use this approach.